flat_mutation_reader.cc

```diff
@@ -912,3 +915,65 @@ class generating_reader final : public flat_mutation_reader::impl {
 flat_mutation_reader make_generating_reader(schema_ptr s, std::function<future<mutation_fragment_opt> ()> get_next_fragment) {
     return make_flat_mutation_reader<generating_reader>(std::move(s), std::move(get_next_fragment));
 }
+
+bool mutation_fragment_stream_validator::operator()(const dht::decorated_key& dk) {
+    // FIXME: Validate fragment position monotonicity
+    //        https://github.com/scylladb/scylla/issues/4804
```

Comment:
Why is this FIXME here? Isn't this done below already?

Reply:
The issue also calls for partition key monotonicity validation (taking into account the partitioner of course).  The comment is wrong (copy-pasted). Will fix.